tutorindigo/templates/indigo/lms/static/sass/xblock/_xblock.scss

```diff
@@ -57,7 +57,7 @@
     .xmodule_display.xmodule_HtmlBlock ul, 
     .xmodule_display.xmodule_StaticTabBlock ol, 
     .xmodule_display.xmodule_StaticTabBlock ul {
-        color: $text-color-d;
+        color: $text-color-d !important;
```

Comment:
AFAIK `!important` css tags are bad practice. Is there a legitimate reason to use them here? Is there any way to avoid them?